command/cp.go

```diff
@@ -128,6 +128,14 @@ var copyCommandFlags = []cli.Flag{
 		Name:  "acl",
 		Usage: "set acl for target: defines granted accesses and their types on different accounts/groups",
 	},
+	&cli.StringFlag{
+		Name:  "src-region",
+		Usage: "set the region of source bucket: only useful when the correct region info cannot be fetched",
```

Comment:
`only useful when the correct region info cannot be fetched` might be misleading. some users may prefer to set `--source-region` (even auto-fetch logic works) since it sends additional API request. How about something like:
```
set the region of source bucket; the region of the source bucket will be automatically discovered if --source-region is not specified
```

Reply:
@sonmezonur does that mean after this pull is merged, if i explicitly set `--source-region` i can save a roundtrip request on each file? 

(for context: I'm doing millions of small files using `run` with `cp`)

Reply:
> @sonmezonur does that mean after this pull is merged, if i explicitly set `--source-region` i can save a roundtrip request on each file?
> 
> (for context: I'm doing millions of small files using `run` with `cp`)


Actually, it does not. `s5cmd` retrieves bucket location during session creation and re-uses these sessions (from global cache) afterward. For each bucket, we are sending [HeadBucket](https://docs.aws.amazon.com/en_us/AmazonS3/latest/API/API_HeadBucket.html) request to determine its region. So if you are copying files to `same bucket`, you can save only one roundtrip request. But if you have multiple destination buckets in your run file, you may expect more.

Reply:
@sonmezonur I agree with you on that. Your statement looks much better.